apk-generator/v2/app/static/js/main.js

```diff
@@ -55,6 +55,10 @@ $dataSourceRadio.change(
     }
 );
 
+$('#aLog').click(function(e) {
+    $('#buildLog').toggle();
```

Comment:
Don't select the element each time `#aLog` is clicked. Select once. Store and re-use.